app/controllers/admin/health/patient_referrals_controller.rb

```diff
@@ -168,6 +176,25 @@ def assign_agency
     end
     # rubocop:enable Style/IfInsideElse
 
+    private def assign_agency_inner(patient_referral, agency_id: nil, care_coordinator_id: nil, nurse_care_manager_id: nil)
+      care_staff_id = care_coordinator_id.presence || nurse_care_manager_id.presence
```

Comment:
what happens if both care_coordinator_id and nurse_care_manager_id are blank?

Reply:
Copying from my comment above to Sandy, there are 3 cases:

1. The PR was previously unassigned, and user is assigning it. In this case, agency is blank and CC or NCM is present. Care staff is used to infer agency to assign to (Tab 1)
2. The PR was previously assigned, and user is re-assigning it to a different agency. In this case, the agency_id is present, and CC/NCM are nil. (Tab 2)
3. The PR was previously assigned, and user is un-assigning it. In this case, all three of agency_id/CC/NCM are nil. (Tab 2)

Reply:
Ok, I _think_ the change below to return if all three are blank (encapsulated in a blank `agency_id` should make it clear this is ok.  Thanks!

Reply:
We don't actually want to return immediately if all three are blank (case 3). In this case we need to perform the unassigment first, which happens with `patient_referral.update`.